pkg/apis/pipeline/v1alpha1/pull_request_resource.go

```diff
@@ -25,8 +25,10 @@ import (
 )
 
 const (
-	prSource       = "pr-source"
-	githubTokenEnv = "githubToken"
+	prSource         = "pr-source"
+	githubTokenField = "githubToken"
+	// nolint: gosec
```

Comment:
what's this about? (might be worth a comment of its own)

Reply:
I can add a comment as a follow-up.
`gosec` thinks that we might be hardcoding credentials in there....
```
I1126 14:29:17.615] pkg/apis/pipeline/v1alpha1/pull_request_resource.go:30: G101: Potential hardcoded credentials (gosec)
I1126 14:29:17.647] 	githubTokenEnv   = "GITHUB_TOKEN"
```

Reply:
oh weird haha